clicmd

```diff
@@ -31,10 +31,14 @@ if [[ "${USER}" != "root" || ! -v GROUP ]]; then
   fi
   GROUP=$(id -Gn | sed -n 's/.*\(%GROUP_REGEX%\)[ ]*.*$/\1/gp')
 fi
+if [[ -z "${GROUP}" ]]; then
+  echo "Can't find group for user ${USER}" >&2
```

Comment:
It seems to me still incorrect.
We didn't find the group that could help to determine the user's role.
But your message seems to me says the user has no any group's membership at all.

Am I nag? ;)

Reply:
well.. ok =)